app/dts/behaviors/to_layer.dtsi

```diff
@@ -9,6 +9,7 @@
         /omit-if-no-ref/ to: to_layer {
             compatible = "zmk,behavior-to-layer";
             #binding-cells = <1>;
+            friendly-name = "Go To Layer";
```

Comment:
We don't call this "Go To Layer" anywhere in the docs, just "To Layer". I think it'd be good to be consistent, do you prefer the former?

Reply:
As I was reading it.. I realized it was a pretty odd name. I'll update to match our current language.